### PR TITLE
[DK-6146] Add new and enhance current git helpers

### PR DESCRIPTION
**Pull request by eytanhanig**

### CODE REVIEW

**Review by abhaysr — APPROVED** [review]

**Comment by abhaysr** [comment]
Looks good to me

File: git/helpers.sh [hunk 1]
```diff
@@ -1,3 +1,15 @@
+#
+# Open file or directory in web browser; defaults to current directory and branch
+# Syntax: git-web [$file_or_dir] [$branch]
+#
+function gitweb() {
+  file=${1:-""}
```
Comment (eytanhanig) [inline git/helpers.sh:6]: Common practice is to **use `lower_case` names for variables that aren't exported to the shell**. Sources:
1. https://stackoverflow.com/a/673940/4472510
2. https://unix.stackexchange.com/a/42849/112657
3. https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html

Comment (eytanhanig) [inline git/helpers.sh:6]: The common convention is to **use `lower_case` names for variables that aren't exported to the shell**. Sources:
1. https://stackoverflow.com/a/673940/4472510
2. https://unix.stackexchange.com/a/42849/112657
3. https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html

**Review by elroyalva — APPROVED** [review]

File: git/helpers.sh [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+  git_branch=${2:-$(git symbolic-ref --quiet --short HEAD)}
+  git_project_root=$(git config remote.origin.url | sed -r "s~(\w+@|https://|ssh://(\w+@)?)([^:^/]*)(:[0-9]+|)[:/](.*)\.git~https://\3\4/\5~")
```
Comment (Pepelo) [inline git/helpers.sh:8]: This line does not work for me. I get:
```
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
```

Comment (Pepelo) [reply git/helpers.sh:8]: I have replaced this with a simpler and working command here: https://github.com/OtoAnalytics/womply-scripts/pull/33/files#diff-58c49966f0357db4f52a7712cfdc31eee5a934c63088568d737f844cc25dec16R91

**Pull request merged**